css/styles.css

```diff
@@ -0,0 +1,147 @@
+/* Style.css */
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+body {
+  font-family: Century Gothic, sans-serif;
+  font-size: 16px;
+}
+
+h1 {
+  font-size: 3rem;
+  padding: 20px;
+  width: 100%;
+  text-align: center;
+
+}
+
+h1 a {
+  background-image: url("http://www.pexels.com/photo/food-raspberries-fruits-7329/");
+  background-size: cover;
+}
+
+h2 {
+  font-size: 1.5rem;
+  padding: 20px;
+  text-align: center;
+  background-color: lightgrey;
+  width: 100%;
+}
+
+.topics li {
+  font-size: 1rem;
+  float: left;
+  width: 25%;
+  margin-top: 15px;
+  margin-bottom: 5px;
+  padding-left: 5px;
+
+}
+
+.aboutmuse {
+  clear: both;
+  padding: 20px;
+  width: 100%;
+}
+
+button {
+  padding: 20px;
+  background-color: lightgrey;
+  font-size: 14px;
+}
+
+.meetmuse {
+  font-size: 1rem;
+  padding: 20px;
+  text-align: center;
+  background-color: slategrey;
+  width: 100%;
+}
+
+.profilewrap {
+  clear: both;
+  padding: 1%;
+  width: 100%;
+  margin: 1%;
+}
+
+footer {
+  padding: 20px;
+  text-align: center;
+  background-color: lightgrey;
+  width: 100%;
+  display: block;
+  clear: both;
+}
+
+.footerbutton button {
+  margin-top: 20px;
+  padding: 10px;
+  background-color: slategrey;
+  font-size: 10px;
+  clear: both;
+  color: white;
+
+}
+
+.profile p {
+  margin-top: 20px;
+  margin-left: 175px;
+  margin-right: 5%;
+  font-size: 1.1rem;
+
+}
+
+.profile {
+  float: left;
```

Comment:
Instead of putting float: left in profile, since it's used repeatedly, try creating a class just for float called align-left or float-left and use it on the tags that need it.